main.go

```diff
@@ -93,6 +94,7 @@ func main() {
 			"Enabling this will ensure there is only one active controller manager.")
 	opts := zap.Options{
 		Development: true,
+		TimeEncoder: zapcore.ISO8601TimeEncoder,
```

Comment:
Changes the timestamp to an actual date time in the operator logs. Vastly improves readability :)